kubernetes/cray-bos/values.yaml.in

```diff
@@ -8,9 +8,12 @@
 #   pullPolicy: "" (default = "IfNotPresent")
 
 boa_image:
-  repository: cray/cray-boa
-  tag: IMAGE_CRAY_BOA_TAG
-
+  repository: artifactory.algol60.net/csm-docker/stable/cray-boa
+  tag: 0.0.0-boa 
+database:
+  image:
+    repository: docker.io/library/redis
```

Comment:
This should be coming from artifactory.algol60.net

Reply:
It is now.